executor/aggfuncs/func_count_distinct.go

```diff
@@ -76,6 +76,25 @@ func (e *countOriginalWithDistinct4Int) AppendFinalResult2Chunk(_ sessionctx.Con
 	return nil
 }
 
+// TODO implement it
+func (e *countOriginalWithDistinct4Int) NewSpillSerializeHelper() *SpillSerializeHelper {
+	return nil
+}
+
+// TODO implement it
+func (e *countOriginalWithDistinct4Int) SerializeForSpill(_ sessionctx.Context, partialResult PartialResult, chk *chunk.Chunk, spillHelper *SpillSerializeHelper) {
```

Comment:
We can use `baseCount` or `baseAggFunc` to implement these funcs instead of implementing them one by one.

Reply:
> We can use `baseCount` or `baseAggFunc` to implement these funcs instead of implementing them one by one.

As to count_distinct, each agg function needs a custom implementation because their agg meta data type are different.